pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,20 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+The programmatically determined link context is a list of elements. All elements in the list must be [included in the accessibility tree][], and have one of the following relationships to the link:
```

Comment:
I have to point out that "the closest ancestor that is a `p`" is not a relationship _stricto sensu_… (the relationship would be "**is** the closest `p` ancestor"…)

Plus, the definition is currently ambiguous because it doesn't specify the size of the list. So, there is no unicity of PDL context and, typically, the empty list is always a PDL context for any link.


---
The PDL context is the list of all elements which are included in the accessibility tree and are either:
- an ancestor of the link with a role of listitem; or
- the closest ancestor of the link that is a `p`; or
- …

should be good.

---
Replace "serve as row/column header for the cell" by "is assigned to the cell containing the link" + link to https://html.spec.whatwg.org/multipage/tables.html#algorithm-for-assigning-header-cells